README.md

```diff
@@ -210,6 +210,92 @@ This app uses [RSpec](http://rspec.info/) for Rails tests.
 
     rake parallel:spec
 
+## Required environment variables for
 
+* These SMTP settings are used to send emails to prisons. Because these are on the Government Secure Intranet (GSI), they must be sent via MessageLabs. You will need to configured them, if you want to try and test email by submiting a booking request locally. Source: `app/mailers/prison_mailer.rb`
+```ruby
+GSI_SMTP_HOSTNAME
+GSI_SMTP_PORT
+GSI_SMTP_DOMAIN
+```
+
+* These SMTP settings are used to send emails to visitors. Because these using SendGrid's SMTP server, they must be sent via SendGrid. You will need to configured them, if you want to try and test email by submiting a booking request locally. Source: `config/environments/production.rb`
+```ruby
+SMTP_HOSTNAME
+SMTP_PORT
+SMTP_DOMAIN
+SMTP_USERNAME
+SMTP_PASSWORD
+```
+
+* This Service Url that is using for session store the host and links in emails. Though for development and test environment is set to `localhost`. Source: `config/environments/production.rb` && `config/initializers/session_store.rb`
```

Comment:
"This URL is used to configure the session store host and to generate links in emails. For development and test" …